modules/tinymce/src/plugins/advlist/test/ts/browser/AdvlistPluginTest.ts

```diff
@@ -45,7 +64,7 @@ UnitTest.asynctest('browser.tinymce.plugins.advlist.AdvlistPluginTest', (success
     expectedSelection: [ 'li:nth-child(1)', 0 ]
   });
 
-  listStyleTest('Apply unordered list style to an unordered list', {
```

Comment:
Do you mean `Apply unordered list style to an ordered list`?

Reply:
Yep, I got this backwards. Thanks for the catch!

Reply:
Fixed